### DISCUSSION TITLE
Another issue with installation

### DISCUSSION THREAD

**Discussion by vasyah999** [linux-surface/linux-surface#663] [Q&A] [answered]
![Screenshot from 2021-12-03 13-34-51](https://user-images.githubusercontent.com/62846149/144588999-5809c457-c43c-42e7-889f-596803e5543b.png

**Comment by qzed** [comment, marked as answer]
That's an issue with APT, see https://github.com/linux-surface/linux-surface/issues/625#issuecomment-962622556 for a workaround.
